package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "colette",
-  "version": "0.34.0",
+  "version": "0.34.1",
```

Comment:
I think it should move to `0.35.0` because this changement can bring regression in a project using `<fieldset>`

Reply:
ok